operator/v1/types_console.go

```diff
@@ -39,6 +39,9 @@ type ConsoleCustomization struct {
 	// documentation URL.
 	// Invalid value will prevent a console rollout.
 	DocumentationBaseURL string `json:"documentationBaseURL,omitempty"`
+	// statuspageID contains ID for statuspage.io page that provides status info about
+	// cluster itself and related services.
+	StatuspageID string `json:"statuspageID,omitempty"`
```

Comment:
Initial thoughts:

```yaml
Console:
  ConsoleSpec:
    // something indicating 3rd party providers of various services.
    // service could be anything, really.
    Providers:
      StatusPageIO: 
        ID:  `12345`
      // example of adding a second provider, such as for analytics
      Google
        // Google provides several services, one being Analytics
        Analytics:
          ID: `6789`
    // customization of the UI specifically, so we should keep this clean
    Customization
      Brand: `online`
```

Reply:
Looking at the `oauth` config for `IdentityProvider` configuration.  Attempting to mirror this, conceptually.
https://github.com/openshift/api/blob/master/config/v1/types_oauth.go#L29

Reply:
I think if we can get our config to feel pretty consistent with an existing config, as well as ensure we can easily add on additional properties in the future, its probably a win.

Reply:
Example:

```yaml
Providers:
  StatusPageIO:
    // may seem overkill right now, but its possible these 
    // providers will add additional attributes in the future.  
    // this lets us easily just add new fields w/o repeating the 
    // StatusPageIO prefix on numerous top level fields.
    ID: 123456
```

Reply:
Unlike the `oauth` providers, which are all `identity providers`, ours may be providers of any number of services.  So I don't think we would structure like an array, we probably need a map. 
https://github.com/openshift/api/blob/master/config/v1/types_oauth.go#L29